svc/FetchService.js

```diff
@@ -67,26 +72,25 @@ export class FetchService {
             delete fetchOpts.acceptJson;
         }
 
-        delete fetchOpts.contentType;
-        delete fetchOpts.url;
 
         // 3) Preprocess and apply params
         if (params) {
-            const paramsStrings = [];
-            Object.entries(params).forEach(v => {
-                const key = v[0],
-                    vals = castArray(v[1]);
-                vals.forEach(val => paramsStrings.push(`${key}=${encodeURIComponent(val)}`));
-            });
-            const paramsString = paramsStrings.join('&');
-
-            if (method === 'POST') {
+            let qsOpts = {arrayFormat: 'repeat', allowDots: true};
+            qsOpts = Object.assign(qsOpts, opts.qsOpts);
```

Comment:
could do this in one line by spreading opts.qsOpts into the new variable with ...}

Reply:
True. Done.